### PR TITLE
Updated code formatting to be consistent between intellij and eclipse.

### PR DESCRIPTION
**Pull request by ericenns**
## Description of changes
Update Eclipse and Intellij code formatting to be consistent.

## Related issue
Link to the GitHub issue this pull request addresses using the `#issuenum` format.  If it completes an issue, use `Fixes #issuenum` to automatically close the issue.

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [ ] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
* [ ] Tests added (or description of how to test) for any new features.
* [ ] User documentation updated for UI or technical changes.

### CODE REVIEW

File: src/main/java/ca/corefacility/bioinformatics/irida/pipeline/results/updater/impl/BioHanselSampleUpdater.java [hunk 1]
```diff
@@ -59,8 +59,9 @@ public BioHanselSampleUpdater(MetadataTemplateService metadataTemplateService, S
 	 * <p>
 	 * Create a bio_hansel {@link MetadataTemplate} for each {@link Project} of the {@link Sample} if one doesn't exist.
 	 *
-	 * @param samples  The sample to update (collection should only have one {@link Sample} object).
-	 * @param analysis Use the results from this {@link AnalysisSubmission} to update the {@link Sample} metadata.
+	 * @param  samples                 The sample to update (collection should only have one {@link Sample} object).
+	 * @param  analysis                Use the results from this {@link AnalysisSubmission} to update the {@link Sample}
+	 *                                 metadata.
```
Comment (deepsidhu85) [inline src/main/java/ca/corefacility/bioinformatics/irida/pipeline/results/updater/impl/BioHanselSampleUpdater.java:64]: With the intellij formatter the metadata is on the same line as the @param above

File: src/main/java/ca/corefacility/bioinformatics/irida/pipeline/results/updater/impl/BioHanselSampleUpdater.java [hunk 2]
```diff
@@ -69,18 +70,16 @@ public void update(Collection<Sample> samples, AnalysisSubmission analysis) thro
 			throw new PostProcessingException(
 					"Expected one sample; got '" + samples.size() + "' for analysis [id=" + analysis.getId() + "]");
 		}
-		final Sample sample = samples.iterator()
-				.next();
+		final Sample sample = samples.iterator().next();
```
Comment (deepsidhu85) [inline src/main/java/ca/corefacility/bioinformatics/irida/pipeline/results/updater/impl/BioHanselSampleUpdater.java:73]: .next(); is on the next line

File: src/main/java/ca/corefacility/bioinformatics/irida/pipeline/results/updater/impl/BioHanselSampleUpdater.java [hunk 3]
```diff
@@ -69,18 +70,16 @@ public void update(Collection<Sample> samples, AnalysisSubmission analysis) thro
...
+		AnalysisOutputFile aof = analysis.getAnalysis().getAnalysisOutputFile(BIO_HANSEL_RESULTS_FILE);
 
 		Path filePath = aof.getFile();
 
 		Map<String, MetadataEntry> stringEntries = new HashMap<>();
 		try {
-			@SuppressWarnings("resource") String jsonText = new Scanner(
-					new BufferedReader(new FileReader(filePath.toFile()))).useDelimiter("\\Z")
+			@SuppressWarnings("resource")
+			String jsonText = new Scanner(new BufferedReader(new FileReader(filePath.toFile()))).useDelimiter("\\Z")
```
Comment (deepsidhu85) [inline src/main/java/ca/corefacility/bioinformatics/irida/pipeline/results/updater/impl/BioHanselSampleUpdater.java:82]: This one is like what is was before the formatting in eclipse

File: src/main/java/ca/corefacility/bioinformatics/irida/pipeline/results/updater/impl/BioHanselSampleUpdater.java [hunk 4]
```diff
@@ -94,14 +93,13 @@ public void update(Collection<Sample> samples, AnalysisSubmission analysis) thro
 				BIO_HANSEL_RESULTS_FIELDS.forEach((key, field) -> {
 					final String formattedField = getNamespacedField(baseNamespace, field);
 					if ((result.containsKey(key)) && (result.get(key) != null)) {
-						String value = result.get(key)
-								.toString();
+						String value = result.get(key).toString();
```
Comment (deepsidhu85) [inline src/main/java/ca/corefacility/bioinformatics/irida/pipeline/results/updater/impl/BioHanselSampleUpdater.java:96]: .toString(); on next line

File: src/main/java/ca/corefacility/bioinformatics/irida/pipeline/results/updater/impl/BioHanselSampleUpdater.java [hunk 5]
```diff
@@ -94,14 +93,13 @@ public void update(Collection<Sample> samples, AnalysisSubmission analysis) thro
...
 						PipelineProvidedMetadataEntry metadataEntry = new PipelineProvidedMetadataEntry(value, "text",
 								analysis);
 						stringEntries.put(formattedField, metadataEntry);
 					} else {
-						logger.warn("bio_hansel output file '" + filePath.toFile()
-								.getAbsolutePath() + "' does not contain expected key '" + key
+						logger.warn("bio_hansel output file '" + filePath.toFile().getAbsolutePath()
```
Comment (deepsidhu85) [inline src/main/java/ca/corefacility/bioinformatics/irida/pipeline/results/updater/impl/BioHanselSampleUpdater.java:101]: .getAbsolutePath() on the next line

**Review by deepsidhu85 — CHANGES_REQUESTED** [review]
Below are the differences in `BioHanselSampleUpdater.java`

File: src/main/java/ca/corefacility/bioinformatics/irida/pipeline/results/updater/impl/BioHanselSampleUpdater.java (hunk 5 above)
Comment (ericenns) [reply src/main/java/ca/corefacility/bioinformatics/irida/pipeline/results/updater/impl/BioHanselSampleUpdater.java]: Fixed in ca919b6

File: src/main/java/ca/corefacility/bioinformatics/irida/pipeline/results/updater/impl/BioHanselSampleUpdater.java (hunk 4 above)
Comment (ericenns) [reply src/main/java/ca/corefacility/bioinformatics/irida/pipeline/results/updater/impl/BioHanselSampleUpdater.java]: Fixed in ca919b6

File: src/main/java/ca/corefacility/bioinformatics/irida/pipeline/results/updater/impl/BioHanselSampleUpdater.java (hunk 3 above)
Comment (ericenns) [reply src/main/java/ca/corefacility/bioinformatics/irida/pipeline/results/updater/impl/BioHanselSampleUpdater.java:83]: Updated intellij format file in ca919b6 to match eclipse formatting.

File: src/main/java/ca/corefacility/bioinformatics/irida/pipeline/results/updater/impl/BioHanselSampleUpdater.java (hunk 1 above)
Comment (ericenns) [reply src/main/java/ca/corefacility/bioinformatics/irida/pipeline/results/updater/impl/BioHanselSampleUpdater.java]: Fixed in ca919b6

**Review by deepsidhu85 — APPROVED** [review]
Looks good. Thanks for getting these to pretty much match each other

**Pull request merged**